src/crate/client/sqlalchemy/__init__.py

```diff
@@ -23,10 +23,24 @@
 from .dialect import CrateDialect
 from .sa_version import SA_1_4, SA_VERSION
 
-# SQLAlchemy 1.3 does not have the `exec_driver_sql` method.
+
 if SA_VERSION < SA_1_4:
+    import textwrap
+    import warnings
+
+    # SQLAlchemy 1.3 is effectively EOL.
+    SA13_DEPRECATION_WARNING = textwrap.dedent("""
+    WARNING: SQLAlchemy 1.3 is effectively EOL.
```

Comment:
```suggestion
    WARNING: SQLAlchemy 1.3 is currently in maintenance status and will soon be EOL.
```

Reply:
I've adjusted the patch accordingly, thanks.